script/build.sh

```diff
@@ -59,12 +59,17 @@ cp -r $DIR/docs ${SRC_DIR}/pfe/portal/
 # Copy the appsody and odo extension into portal. The zip file must have a version number e.g. codewind-appsody-extension-0.3.0.zip
 # in order for it to be accepted as a valid extension
 echo -e "\n+++   DOWNLOADING EXTENSIONS   +++\n";
-mkdir -p ${SRC_DIR}/pfe/extensions
-rm -f ${SRC_DIR}/pfe/extensions/codewind-appsody-extension-*.zip
+#mkdir -p ${SRC_DIR}/pfe/extensions
+#rm -f ${SRC_DIR}/pfe/extensions/codewind-appsody-extension-*.zip
 rm -f ${SRC_DIR}/pfe/extensions/codewind-odo-extension-*.zip
-curl -Lo ${SRC_DIR}/pfe/extensions/codewind-appsody-extension-9.9.9999.zip http://download.eclipse.org/codewind/codewind-appsody-extension/master/latest/codewind-appsody-extension-9.9.9999.zip
+#curl -Lo ${SRC_DIR}/pfe/extensions/codewind-appsody-extension-9.9.9999.zip http://download.eclipse.org/codewind/codewind-appsody-extension/master/latest/codewind-appsody-extension-9.9.9999.zip
 curl -Lo ${SRC_DIR}/pfe/extensions/codewind-odo-extension-9.9.9999.zip http://download.eclipse.org/codewind/codewind-odo-extension/master/latest/codewind-odo-extension-9.9.9999.zip
 
+# In case of ppc64le, download 'appsody' using ppc64le specific url and build 'appsody-controller' from source
+if [ "$ARCH" == "ppc64le" ]; then
+  $DIR/script/build-appsody-controller.sh
+fi
```

Comment:
I think based on the comment from @neeraj-laad https://github.com/eclipse/codewind/pull/1639#issuecomment-571989188 that support for Appsody in Codewind on ppc64le should probably be delivered when Appsody has full ppc64le support.
This PR will probably be much simpler if it just provides Codewind support for ppc64le without support for the Appsody extension.